src/SFML/Window/Win32/WglContext.cpp

```diff
@@ -376,6 +386,11 @@ int WglContext::selectBestPixelFormat(HDC deviceContext, unsigned int bitsPerPix
                 }
             }
         }
+
+        formatCache.try_emplace(
+            std::make_tuple(bitsPerPixel, settings),
+            bestFormat
+        );
```

Comment:
Why did you pick `try_emplace` compared to `emplace`? At this point, we should be sure that there is no such format in the cache. 

I would rewrite as:

```suggestion
        [[maybe_unused]] const auto [it, emplaced] = formatCache.emplace(
            std::make_tuple(bitsPerPixel, settings),
            bestFormat
        );
        
        assert(emplaced);
```